src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,39 @@
+package racingcar.domain
+
+import java.lang.IllegalArgumentException
+
+class Cars(
+    val cars: List<Car>,
+    private val strategy: MovingStrategy = RandomStrategy()
+) {
+
+    companion object {
+        fun from(carNames: List<String>): Cars {
+            return Cars(
+                carNames.map { Car(it)}
+            )
+        }
+    }
+
+    fun moveAll() {
+        for (car in cars) {
+            if (strategy.isMovable()) {
+                car.moveForward(strategy)
+            }
+        }
+    }
```

Comment:
```suggestion
        cars.map { it.moveForward(strategy) }
```

이렇게도 되겠네요. 적절한 사용법인지는.. :)

Reply:
<img width="878" alt="image" src="https://user-images.githubusercontent.com/46641538/198763117-225d54d7-6d91-4bee-94da-76e46294a009.png">

자바에서는 map이 중간연산이었던 것 같은데 코틀린에서는 그 자체로 쓸 수 있나보네요. 신기합니당